index.html

```diff
@@ -0,0 +1,393 @@
+                                <div class="row card-contents">
+
+                                    <div class="col-3">
+                                        <img src="./images/images/program_icon_04.png" alt="">
+                                    </div>
+
+
+                                    <div class="col-4">
+                                        <h3>Workshop</h3>
+                                    </div>
+
+                                    <div class="col-5">
+                                        <p>We have time to share our thoughts and opinions with experts by topic.</p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="col-11">
+                                <div class="row card-contents">
+
+                                    <div class="col-3">
+                                        <img src="./images/images/program_icon_05.png" alt="">
+
+                                    </div>
+
+                                    <div class="col-4">
+                                        <h3>cc party</h3>
+                                    </div>
+
+                                    <div class="col-5">
+                                        <p>Create opportunities to freely network with other young farmers in West Africa.</p>
+                                    </div>
+                                </div>
+                            </div>
+
+                        </div>
+                        <div class="row">
+                            <div class="col-1"></div>
+                            <div class="col-10 join_summit">
+                                <center>Join summit 2023</center>
+                            </div>
+
+                        </div>
+
+                    </div>
+
+
+                </div>
+                <div class="row down_part">
+                    <div class="col-12">
+
+                        <center>
+                            <span>
+                                <a href="##">See whole program </a>
+                            </span>
+                        </center>
+
+                    </div>
+                </div>
+            </section>
+
+            <!--Down section-->
+            <section class="container down_section" id="desktop_speakers">
+
+      <!--contents of speaker cards coming from DOM in javascript-->
+                
+
+            </section>
+            <!--Mobile speaker parts-->
+            <!--Down section-->
+            <section class="container down_section" id="mobile_skpeakers">
+
+    
+            </section>
+            <!-- partners -->
+            <div class="w-100 partners">
+                <img src="./images/images/farm_footer.png" alt="">
+
+            </div>
+
+            <!-- footer -->
+            <div class="container footer_home">
+                <div class="row">
```

Comment:
- [ ] I would like to wrap up all footer elements inside the footer tag instead of a semantic reason because A `<footer>` typically contains information about the author of the section, copyright data, or links to related documents. :+1: